README.md

```diff
@@ -104,39 +104,68 @@ For reference, here is a truth table for the expression A && !B. You can fill o
 
 | A     | B     | !B    | A && !B | !A && !B |
 | ----- | ----- | ----- | ------- | -------- |
-| true  | true  | false | false   |          |
-| false | true  | false | false   |          |
-| true  | false | true  | true    |          |
-| false | false | true  | false   |          |
+| true  | true  | false | false   | false    |
+| false | true  | false | false   | false    |
+| true  | false | true  | true    | false    |
+| false | false | true  | false   | true     |
 
 ## Problem Five
 
 Create a truth table for the expression !(A || B).
+| A     | B     | !(A \|\| B) |
+|-------|-------|-------------|
+| TRUE  | TRUE  | FALSE       |
+| FALSE | TRUE  | FALSE       |
+| TRUE  | FALSE | FALSE       |
+| FALSE | FALSE | TRUE        |
+
 
 ## Problem Six
 
 Write a step-by-step evaluation for the following expression ([remember order of operations](https://www.mathsisfun.com/operation-order-pemdas.html)): `2 + 3 * 2 + 1`.
 For reference, here is a exp of a step-by-step evaluation:
 
+
 ```js
 1 + 2 + 3 + 4
 3 + 3 + 4
 6 + 4
 10
 ```
 
+```js
+//(3 * 2) + 1 + 2
+//6 + 1 + 2
+//6 + 3
+//9
+```
+
+
 ## Problem Seven
 
+```js
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
+//(4/2) + (8/4)
+//2 + 2
+//4
 
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
-
+//caterpillar
```

Comment:
The steps of this are a little different. Starting from step 2. 
'catter' + 'pi' + 'llar' 
'caterpi' + 'llar' 

Then 
caterpillar